### PR TITLE
Minor lang fixes

### PR DESCRIPTION
**Pull request by kumquat-ir**
**What:**
Fixes some minor issues with localization and tooltips

**How solved:**
Restored "not enough energy!" chat message when trying to use a prospector with too little energy
Fixes light grey chemical dye localization in en_us (ru_ru and zh_cn had the correct keys already)
Clarify tooltips on cfoam sprayer and block breakers

### CODE REVIEW

File: src/main/resources/assets/gregtech/lang/en_us.lang [hunk 1]
```diff
@@ -2206,6 +2206,7 @@ metaitem.prospector.mode.ores=Ore Prospection Mode
 metaitem.prospector.mode.fluid=Fluid Prospection Mode
 metaitem.prospector.tooltip.ores=Scans Ores in a %s Chunk Radius
 metaitem.prospector.tooltip.fluids=Scans Ores and Fluids in a %s Chunk Radius
+behavior.prospector.not_enough_energy=Not Enough Energy!
```
Comment (ALongStringOfNumbers) [inline src/main/resources/assets/gregtech/lang/en_us.lang:2209]: Why does this not have the `Analyzing Failed:` portion like the same key in zh_cn?

Comment (kumquat-ir) [reply src/main/resources/assets/gregtech/lang/en_us.lang:2209]: That's exactly how it was before I removed the scanner lang keys without checking if anything else used them

File: src/main/resources/assets/gregtech/lang/en_us.lang [hunk 2]
```diff
@@ -2597,7 +2598,7 @@ gregtech.machine.block_breaker.lv.name=Basic Block Breaker
 gregtech.machine.block_breaker.mv.name=Advanced Block Breaker
 gregtech.machine.block_breaker.hv.name=Advanced Block Breaker II
 gregtech.machine.block_breaker.ev.name=Advanced Block Breaker III
-gregtech.machine.block_breaker.tooltip=Mines block on front face and collects its drops
+gregtech.machine.block_breaker.tooltip=Mines block on front face and collects its drops/nOnly works when powered with Redstone
```
Comment (TechLord22) [inline src/main/resources/assets/gregtech/lang/en_us.lang:2601]: Can you add this to the item collector as well? We found this was also the case for that one.

**Review by TechLord22 — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/gregtech/lang/en_us.lang (hunk 2 above)
Comment (kumquat-ir) [reply src/main/resources/assets/gregtech/lang/en_us.lang:2601]: That seems to already be in the tooltip

**Review by TechLord22 — APPROVED** [review]

**Pull request merged**